applications/tasks/src/knm_number_crawler.erl

```diff
@@ -98,6 +98,9 @@ handle_call(_Request, _From, State) ->
 handle_cast('stop', State) ->
     lager:debug("crawler has been stopped"),
     {'stop', 'normal', State};
+handle_cast({'$proxy_stop', Reason}, State) ->
+    lager:debug("~p stopping with reason: ~p", [?MODULE, Reason]),
+    {stop, Reason, State};
```

Comment:
plz tickie all the atoms

Reply:
`handle_cast({'$proxy_stop', Reason}, State) ->` are these really necessary ?